### PR TITLE
[SYTO-1223] Mortar Copy Changes

### PR DESCRIPTION
**Pull request by sebasrgl**
https://jira.natgeo.com/browse/SYTO-1223

__How to__
1. ``gulp serve``
2. Check AC met according to the document.
3. Merry Christmas :christmas_tree: :santa:

### CODE REVIEW

File: src/icons/_icon-examples.html [hunk 1]
```diff
@@ -1,21 +1,14 @@
...
-    <p class="mt3_subh4">The following are examples of icons placed next to labels.</p>
+    <h3 class="mt3_heading--h4--2">Using Icons Labels</h3>
+    <p class="mt3_subh4">See the following examples for placing a label to the
+    left of, to the right of, above, or below an icon.</p>
   </div>
 
-  <!-- Icon left example -->
-  <div class="mt3_col-gut">
+<!-- Icon left and right example -->
+  <div class="mt3_col-lg-8 mt3_col-lg-offset-1 mt3_col-gut">
```
Comment (krlospelaez) [inline src/icons/_icon-examples.html]: Remove `mt3_col-lg-8 mt3_col-lg-offset-1`

**Comment by krlospelaez** [comment]
LGTM :+1:

File: src/site/_partials/home/getting_started.html [hunk 2]
```diff
@@ -19,28 +19,32 @@ <h2 class="mt3_heading--h4--2">Install Mortar with npm</h2>
...
-    Mortar's SCSS and CSS are included in the package.<br />
-    Use precompiled bundle if:<br />
-    <br />
-    - You want start developing a.s.a.p.<br />
-    - You don't want to add task to compile sass files.<br />
-    - You don't want to worry about digging sass files.<br />
-    - You're ok adding all mortar's css on your project.
+  <p class="mt3_subh1">
+    Mortar's SCSS and CSS are included in the package. Use precompiled
+    <br/> bundle if:
```
Comment (pmourelle) [inline src/site/_partials/home/getting_started.html]: Lose the `<br>`

File: src/site/_partials/home/philosophy.html [hunk 3]
```diff
@@ -3,21 +3,26 @@
     <h3 class="mt3_heading--h4--2">Design Philosophy</h3>
 
     <p class="mt3_subh4">
-      The styles contained in Mortar reflect National Geographic's overarching design philosophy. We pride
-      ourselves on stunning, world-class visual storytelling. By keeping design elements minimal, we are able
-      to allow our content to drive the visual presentation of our digital products. The design should never feel
-      heavy or overpowering.
+      The styles contained in Mortar reflect National Geographic's overarching
+      design philosophy. We pride ourselves on stunning, world-class visual
```
Comment (pmourelle) [inline src/site/_partials/home/philosophy.html]: No newlines. That depends of the styling not the wording ;)

File: src/site/_partials/home/philosophy.html [hunk 4]
```diff
@@ -3,21 +3,26 @@
...
+       href="https://brandfolder.com/organizations/national-geographic">"Further"</a> with our
+      stories.
     </p>
 
     <p class="mt3_subh4">
-      Refer to the pages in this site to create a repeatable, visual language for users interacting with the
-      National Geographic brand on any digital platform. Our color palette, typographic styles, icon library, and
-      page grid help to create consistent patterns, clean lines, and beautiful displays for our content.
+      Refer to the pages in this site to create a repeatable, visual language for users
+      interacting with the National Geographic brand on any digital platform.
```
Comment (pmourelle) [inline src/site/_partials/home/philosophy.html]: Same.

File: src/site/color/index.html [hunk 5]
```diff
@@ -5,9 +5,11 @@
 
 <div class="pagewrapper mt3_row mt3_col-gut sectionpadding">
   <div class="mt3_col-lg-10 mt3_row-gut-2">
-    <h1 class="mt3_heading--h1--2"> Color Palette </h1>
+    <h1 class="mt3_heading--h1--2">Color</h1>
     <p class="mt3_page-description">
-      National Geographic&apos;s visual brand is driven by stunning photography. Color should be used sparingly to indicate special importance, calls to action, or branding.
+      National Geographic&apos;s visual brand is driven by stunning
```
Comment (pmourelle) [inline src/site/color/index.html]: No newlines.

File: src/site/icons/index.html [hunk 6]
```diff
@@ -5,29 +5,63 @@
 <div class="mt3_row pagewrapper sectionpadding">
   <div class="mt3_col-lg-10 mt3_row-gut-2 mt3_col-gut">
     <h1 class="mt3_heading--h1--2">Icons</h1>
-    <p class="mt3_page-description mt3_row-gut-3">Icons create a visual language for navigation and user interaction.</p>
+    <p class="mt3_page-description mt3_row-gut-3">Icons create a visual language
+    for navigation and user interaction. Click below to download the icon set to
```
Comment (pmourelle) [inline src/site/icons/index.html]: No newlines

**Pull request merged**